runtime/centrifuge/src/xcm.rs

```diff
@@ -55,18 +56,27 @@ use super::{
 pub struct XcmConfig;
 impl xcm_executor::Config for XcmConfig {
 	type AssetClaims = PolkadotXcm;
+	type AssetExchanger = ();
+	type AssetLocker = ();
 	// How to withdraw and deposit an asset.
 	type AssetTransactor = FungiblesTransactor;
 	type AssetTrap = PolkadotXcm;
 	type Barrier = Barrier;
+	type CallDispatcher = RuntimeCall;
+	type FeeManager = ();
 	type IsReserve = MultiNativeAsset<AbsoluteReserveProvider>;
 	type IsTeleporter = ();
-	type LocationInverter = LocationInverter<Ancestry>;
+	type MaxAssetsIntoHolding = ConstU32<64>;
+	type MessageExporter = ();
 	type OriginConverter = XcmOriginToTransactDispatchOrigin;
+	type PalletInstancesInfo = crate::AllPalletsWithSystem;
```

Comment:
It is since this is in the `xcm.rs` module and it's referring to objects present in `lib.rs`. I can import that at the top of the module if you prefer that?